applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -250,6 +250,26 @@ endif
 
 if BT_CENTRAL
 
+if DESKTOP_USB_ENABLE
+config DESKTOP_BLE_MANAGE_CI
+	bool "Enable managing connection intervals after USB state events"
+	default y if DESKTOP_HID_DONGLE
+	help
+	  Enable to manage connection interval for BT peers after USB state events.
+	  This function will set larger connection interval for all peers during
+	  USB suspended mode to reduce power concumption of USB dongle.
+	  After USB will be activated - connection interval will be set to
+	  previous value.
+
+config DESKTOP_BLE_CI_SUSPENDED_MS
+	int "BLE connection interval during USB_STATE_SUSPENDED"
+	default 100
+	range 10 100
+	depends on DESKTOP_BLE_MANAGE_CI
+	help
+	  Connection interval for all connected peers during USB_STATE_SUSPENDED.
```

Comment:
btw. Should we add information in ble_conn_params module doc to inform about new behavior ?

Reply:
This PR needs to be green today/tomorrow. Let's not open another pandora box. We can do a follow up in doc after this is merged. At the same time we can mentioned the change in release notes.